plugins/modules/podman_import.py

```diff
@@ -127,29 +135,65 @@ def load(module, executable):
         module.fail_json(msg="Could not parse JSON from image %s: %s" % (image_name, e))
     return changed, out, err, info, command
 
+def volume_load(module, executable):
+    changed = False
+    command = [executable, 'volume', 'import', module.params['volume'], module.params['src']]
+    src = module.params['src']
+    if os.path.exists(src):
```

Comment:
We can let Podman handle this, in line 147 below it will fail when not finding the requested volume, with appropriate message.